### PR TITLE
301 web app java tomcat

### PR DESCRIPTION
**Pull request by DarqueWarrior**
This is a template that enables Java and Tomcat on an Azure Web app.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@DarqueWarrior__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by DarqueWarrior** [comment]
I think I have corrected the metadata issues.

**Comment by singhkay** [comment]
@DarqueWarrior As part of the template file, I don't see where Java or Tomcat are enabled ?

**Comment by DarqueWarrior** [comment]
Looks like I checked in the wrong version. I will correct. Sorry about that.

**Comment by singhkay** [comment]
@DarqueWarrior  Can you rename this one to 201-web-app-java-tomcat before merging? Also, I'm removing the CLA required tag since your're MS FTE but I also sent you an email about how to avoid the bot prompting for CLA since you're an FTE

**Comment by DarqueWarrior** [comment]
Sure

-----Original Message-----

File: 301-web-app-java-tomcat/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Create a web app on Azure with Java 8 and Tomcat 8 enabled.
+This template creates a web app on azure with Java 8 and Tomcat 8 enabled allowing you to run Java applications in Azure.  Template was authored by [Donovan Brown](http://twitter.com/DonovanBrown) of Microsoft.
```
Comment (singhkay) [inline 301-web-app-java-tomcat/README.md]: Please also include a deploy to azure button in the readme

**Comment by DarqueWarrior** [comment]
I created the Deploy Button with the URL that will be correct once it is merged into Master.

**Comment by singhkay** [comment]
@DarqueWarrior Thanks! lgtm, merged

**Comment by DarqueWarrior** [comment]
Awesome!  Thanks for all your help getting this merged.

**Pull request merged**